pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>com.internetitem</groupId>
     <artifactId>logback-elasticsearch-appender</artifactId>
-    <version>1.6-eocs</version>
+    <version>1.7-eocs</version>
```

Comment:
Please make that `1.6-eocs-2` or `1.6-eocs.2` (we are still building on the 1.6 version of the upstream project).

(Yes, we should have started with `eocs-0`/`eocs.0` or `eocs-1`/`eocs.1` …)